nylas/utils.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import division
+from datetime import datetime
+
+
+def timestamp_from_dt(dt, epoch=datetime(1970, 1, 1)):
+    """
+    Convert a datetime to a timestamp.
+    https://stackoverflow.com/a/8778548/141395
+    """
+    delta = dt - epoch
+    # return delta.total_seconds()
+    return delta.seconds + delta.days * 86400
+
+
+def timestamped_data(data, datetime_attrs):
```

Comment:
Could we rename this? It makes me think we're timestamping all data. Maybe like "convert_datetimes_to_timestamps" or something similar?